cli/pkg/kctrl/cmd/package/installed/create_or_update.go

```diff
@@ -89,15 +97,24 @@ func NewInstallCmd(o *CreateOrUpdateOptions, flagsFactory cmdcore.FlagsFactory)
 		Use:   "install",
 		Short: "Install package",
 		RunE:  func(_ *cobra.Command, args []string) error { return o.RunCreate(args) },
+		Example: `
+# Install a package
+kctrl package install -i <package-install-name> -p <package-name> --version <package-version>
```

Comment:
examples should be concrete to let the user know whats going on. instead of above (may be find cert-manager as example from TCE?)

kctrl package install -i my-pkg -p pkg.name.com --version 1.0.0

Reply:
Done.